projects/exui-common-lib/src/lib/gov-ui/components/hmcts-identity-bar/hmcts-identity-bar.component.html

```diff
@@ -3,11 +3,11 @@
     <div class="hmcts-identity-bar__container">
 
         <div class="hmcts-identity-bar__details">
-        
-            <span class="hmcts-identity-bar__title">{{value}}</span>
-        
-        </div>        
+
```

Comment:
@johnbenjamin-hmcts - can remove the extra whitespace here

Reply:
I have now updated the code.